src/restaurants.json

```diff
@@ -4,55 +4,90 @@
     "name_res": "Mediterranean Delight",
     "chef": "Elena Rodriguez",
     "top_dishes": [
-      { "dish": "Grilled Octopus", "url": "https://example.com/octopus" },
-      { "dish": "Seafood Paella", "url": "https://example.com/paella" },
-      { "dish": "Lemon Herb Chicken", "url": "https://example.com/chicken" }
+      { "dish": "Grilled Octopus", "url": "./assets/images/food/chicken.jpeg" },
```

Comment:
In my opinion dishName would be more descriptive and clear key name for the dish name (: